en/micro-integrator/docs/references/connectors/mongodb-connector/mongodb-connector-example.md

```diff
@@ -0,0 +1,257 @@
+
+1.  Create a new integration project named `MongodbConnector`. Be sure to enable a Connector Exporter.
+
+    <img src="../../../../assets/img/connectors/mongodb-conn-3.png" title="Create project" width="500" alt="Create project"/>
+
+2.  Right click on the created Integration Project and select, -> **New** -> **Rest API** to create the REST API.
+
+    <img src="../../../../assets/img/connectors/adding-an-api.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
+
+3.  Provide the API name as `MongoConnector` and the API context as `/mongodbconnector`.
+
+4.  First we will create the `/insertmany` resource. This API resource will insert documents into the MongoDB database.<br/>
+    Right click on the API Resource and go to **Properties** view. We use a URL template called `/insertmany` as we have two API resources inside single API. The method will be `Post`.
+
+    <img src="../../../../assets/img/connectors/mongodb-conn-4.png" title="Adding the API resource." width="800" alt="Adding the API resource."/>
+
+5.  Drag and drop the 'insertMany' operation of the MongoDB Connector to the Design View as shown below.
+
+    <img src="../../../../assets/img/connectors/mongodb-conn-5.png" title="Adding the insert many operation." width="800" alt="Adding the insert many operation."/>
+
+6.  Create a connection from the properties window by clicking on the '+' icon as shown below.
+
+    Following values can be provided to connect to MongoDB database. <br/>
+
+    - Connection Name - connectionURI
+    - Connection Type - URI
+    - Connection URI - mongodb+srv://server.example.com/?connectTimeoutMS=300000&amp;authSource=aDifferentAuthDB
+    - Database - TestDatabase
+
+    <img src="../../../../assets/img/connectors/mongodb-conn-6.png" title="Adding the connection." width="800" alt="Adding the connection."/>
+
+7.  After the connection is successfully created, select the created connection as 'Connection' from the drop down in the properties window.
+
+    <img src="../../../../assets/img/connectors/mongodb-conn-7.png" title="Selecting the connection." width="800" alt="Selecting the connection."/>
+
+8.  Next, provide the expressions as below to the following properties in the properties window to obtain respective values from the JSON request payload.
+
+    - Collection - json-eval($.collection)
+    - Documents - json-eval($.documents)
+
+9.  Drag and drop the [Respond Mediator](https://ei.docs.wso2.com/en/latest/micro-integrator/references/mediators/respond-Mediator/) to respond the response from inserting documents as shown below.
+
+    <img src="../../../../assets/img/connectors/mongodb-conn-8.png" title="Adding the respond mediator." width="800" alt="Adding the respond mediator."/>
+
+10. Create the next API resource, which is `/find` by dragging and dropping another API resource to the design view. This API resource will find all the documents matching the find query given by the user. This will also be a `POST` request.
```

Comment:
```suggestion
10. Create the next API resource (which is `/find`) by dragging another API resource to the Design view. This API resource will find all the documents matching the find query given by the user. This will also be a `POST` request.
```